src/main/business/applicationFacade.js

```diff
@@ -336,6 +350,45 @@ class ApplicationFacade{
         return this.userController.updateUser(data);
     }
 
+
+    addManualMetaQuestionToExam(data) {
```

Comment:
add a call to `validateParameters` so we know what is the received data
maybe @OfekN11 can do it for you

Reply:
This should be (and will be) done at the business level, at the implementation.

Thats due to the possibilty that more function will call it.

The validation should be done by the ones who use the data.